pkgs/development/python-modules/feedparser/default.nix

```
@@ -1,19 +1,25 @@
 { stdenv
 , buildPythonPackage
 , fetchPypi
+, isPy3k
+, sgmllib3k
+, tox
 }:
 
 buildPythonPackage rec {
   pname = "feedparser";
-  version = "5.2.1";
+  version = "6.0.1";
+
+  disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1ycva69bqssalhqg45rbrfipz3l6hmycszy26k0351fhq990c0xx";
+    sha256 = "0k77c66ia4m64kmdfa2yv8z8887hhwx91pv8b4s2ix23mbf8xa3c";
   };
 
-  # lots of networking failures
-  doCheck = false;
+  propagatedBuildInputs = [ sgmllib3k ];
+
+  checkPhase = "python tests/runtests.py";
```

```suggestion
  checkPhase = "${python.interpreter} tests/runtests.py";
```
support alternate interpreters